staging/src/k8s.io/apiserver/pkg/server/options/authorization.go

```diff
@@ -238,6 +246,9 @@ func (s *DelegatingAuthorizationOptions) getClient() (kubernetes.Interface, erro
 	if s.CustomRoundTripperFn != nil {
```

Comment:
same thing about being unsure whether this should go before or after the tracing wrappers

Reply:
same change made to the WithCustomRoundTripper function.